### DISCUSSION TITLE
How to search inside a same collection relationship

### DISCUSSION THREAD

**Discussion by 0x6775737461766f** [pocketbase/pocketbase#3764] [Q&A]
So, i'm making a social media, and in the "users" collection i added two fields, "followers", and "following", and when i try to get the users, all the info comes through, except for these two relationship columns which return an empty array even if the user has others id on these fields:
`const resultList = await pb.collection('users').getFullList();`

Result:
`{
    avatar: '81849253_7f87_4409_a5ad_63c3f7985bb0_yj0nPjYQlz.jpeg',
    collectionId: '_pb_users_auth_',
    collectionName: 'users',
    created: '2023-11-16 16:17:09.028Z',
    emailVisibility: false,
    followers: [],
    following: [],
    id: 'gukzard2mfz44qs',
    name: 'Test ',
    updated: '2023-11-16 18:05:10.248Z',
    username: 'erick18',
    verified: true
  }`

Is it possible to achieve this? I even tried using expand and filter but none of which helped, and also my only API Filters to list and view are that the user needs to send an id
`@request.auth.id != ""`

**Comment by ganigeorgiev** [comment]
I'm not sure I follow. Does your user record actually have any values set for those fields?
If your users have assigned `followers` and `following` values but they are not returned in the response then make sure that there is no cache involved (_if you use Next.js note that they use a custom `fetch` implementation that comes with cache enabled by default_).

For expanding relations you could check https://pocketbase.io/docs/working-with-relations/#expanding-relations.

If that doesn't help, please paste your collections configuration as json (_Admin UI > Settings > Export collections_) and elaborate on the desired behavior.
